app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+
+<%= stylesheet_link_tag 'registration' %>
+<section class="user-registration">
+<div class="reg-container">
+  <h3 class="reg-header">Sign up</h3>
+  <div class="sign-up-form">
+    <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+      <%= render "devise/shared/error_messages", resource: resource %>
+      <div class="field">
+        <%= f.label :name %><br />
+        <%= f.text_field :name, autofocus: true, autocomplete: "email" %>
+      </div>
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password %>
+        <% if @minimum_password_length %>
+        <em>(<%= @minimum_password_length %> characters minimum)</em>
+        <% end %><br />
+        <%= f.password_field :password, autocomplete: "new-password" %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password_confirmation %><br />
+        <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+      </div>
+
+      <div class="actions">
+        <%= f.submit "Sign up" %>
+      </div>
+    <% end %>
+  </div>
+  <%= render "devise/shared/links" %>
+  </div>
+</section>
```

Comment:
Good job on styling the login and registration pages, but:

- [ ] When I try to register a new user, the app crashes and gives me the errors shown in the screenshot below. Kindly work on making registration work as expected and then the registered user is able to log in and view their profile and more...

<img width="1789" alt="Screenshot 2023-03-10 at 8 16 06 AM" src="https://user-images.githubusercontent.com/50941074/224262644-acdd7a60-5850-49ce-9c24-ba12b168296c.png">